File: Python_Workspace/Python_Tasks/15_06/7.py
#Prime number
a=int(input("Enter the number: "))
count=0
for i in range (1,a+1):
  if(a%i==0):
    count+=1
if(count==2):
  print(a,"is a Prime Number")
else:
  print(a,"is not a Prime Number")

File: C_Workspace/C_CheckList/Digits_Manipulation/7.c
//C Program to print occurrence of a particular digit in a number.
#include <stdio.h>

int main()
{
    int n, digit, count = 0;
    
    printf("Enter the number: ");
    scanf("%d", &n);
    printf("Enter the digit to count: ");
    scanf("%d", &digit);
    
    int temp = n;
    while(temp > 0)
    {
        if(temp % 10 == digit)
        {
            count++;
        }
        temp = temp / 10;
    }
    
    printf("Digit %d occurred %d times in %d.", digit, count, n);
    return 0;
}
